packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -134,18 +134,15 @@
   <div class="top-nav">
     {#if $appStore.initialised}
       <div class="topleftnav">
-        <span class="back-to-apps">
-          <Icon
-            size="S"
-            hoverable
-            name="BackAndroid"
-            on:click={() => $goto("../../portal/apps")}
-          />
-        </span>
+        <a href={$url("../../portal/apps")} class="linkWrapper back-to-apps">
+          <Icon size="S" hoverable name="BackAndroid" />
+        </a>
         <Tabs {selected} size="M">
           {#each $layout.children as { path, title }}
             <TourWrap stepKeys={[`builder-${title}-section`]}>
               <Tab
+                link
```

Comment:
I'm still inclined to pull the anchors out of the `Tab` core code and make them a completely optional slot.

Something like this?
```html

<Tab
    link
    quiet
    selected={$isActive(path)}
    on:click={topItemNavigate(path)}
    title={capitalise(title)}
    id={`builder-${title}-tab`}
  >
  <span slot="label">
   <!-- Get the same ux with just a slot? --/>
    <a href={$url(path)} class="spectrum-Tabs-item link">
      {capitalise(title)}
    </a>
  </span>
</Tab>

```

Reply:
I think this would be a good approach, but though the onClick event does bubble from the slotted anchor up into the nav onClick handler, it doesn't seem like it's actually having time to complete before the default anchor onClick behaviour changes the page. 

This means the storing off your selected/component when navigating between sections doesn't work properly, could obviously fix that by overriding the default onClick behaviour of the slotted anchor, but then any consumer of Tabs that wants links need to implement that behaviour. This makes me lean more towards the current implementation.

Reply:
Fair enough, I had a quick attempt at the scenario there and I can see what you mean now. We would need find another way to balance the single app behaviour and the native handlers 👍